### DISCUSSION TITLE
Support for Coffee machine

### DISCUSSION THREAD

**Discussion by Chimpepe** [QuickSander/homebridge-mieleathome#57] [Ideas]
Is it possible to add support for Miele Coffee machines? 

In my Case it's a Miele CM7750, it's also Miele@Home ready.

Greetings
Dawid

**Comment by dennisbekker** [comment]
If thats possible, i want to to power on my Miele Coffee machine when some sensors are sense motion. so if you can add this this would be great. i am using the CVA7845. i can use al functions power etc with the Miele Home app

**Reply by dennisbekker** [reply]
Wow, update installed can now power the coffee machine off and on :-) thanks!!

**Comment by Humuku** [comment]
Hello and thank you very much for your work. I would be happy, if you could make it possible to start one of the saved drinks on the machine.

**Comment by Siff3D** [comment]
Just to add that the plugin works great with **CM 6160** (for unknown reasons, the CM 6160 doesn't have the settings to automatically turn on/off, which were available on the CM 6150!)

Even though the Miele@Home app also doesn't provide the option to turn on/off the machine, the plugin works and the scheduled turn on/off works with a simple automation in the Home app.

Thank you!
